src/@context/Asset.tsx

```diff
@@ -10,19 +10,19 @@ import React, {
 import { Asset, Config, LoggerInstance, Purgatory } from '@oceanprotocol/lib'
 import { CancelToken } from 'axios'
 import { retrieveDDO } from '@utils/aquarius'
-import { getPrice } from '@utils/subgraph'
 import { useWeb3 } from './Web3'
 import { useSiteMetadata } from '@hooks/useSiteMetadata'
 import { useCancelToken } from '@hooks/useCancelToken'
 import { getOceanConfig, getDevelopmentConfig } from '@utils/ocean'
+import { getConsumeDetails } from '@utils/consumeDetails'
 
 interface AssetProviderValue {
   isInPurgatory: boolean
   purgatoryData: Purgatory
   ddo: Asset
   title: string
   owner: string
-  price: BestPrice
+  consumeDetails: ConsumeDetails
```

Comment:
in general I would suggest we stop using `consume` also in our code base, and replace it with `access` so in this case `accessDetails`.

Reason being that on the UI side we never say "Consume", I mean, who would speak like that, nobody says "I'm gonna consume that file.". "I'm gonna access that file" much more so. Using "access" as the umbrella term for download/compute/(streaming later on?) mirrors how we communicate to end users in our UI.

Reply:
fair enough, more refactor yay